app/backend/backend/migrations/0004_customadmin_report_member_doctor.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.1.2 on 2022-11-20 08:19
+
+import datetime
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('backend', '0003_remove_doctor_customuser_ptr_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CustomAdmin',
+            fields=[
+                ('admin_username', models.CharField(max_length=50, primary_key=True, serialize=False, unique=True)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.CreateModel(
```

Comment:
migrations file include Report model and related stuff in other migrations file. We didn't work with report in this PR but it occurs, this may not cause a failure or pose a danger but I wanted to point it out.

Reply:
It is included in past migration files.